### PR TITLE
Eslint setup

### PR DESCRIPTION
**Pull request by eszthoff**

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+        "experimentalDecorators": 0,
+        "react/jsx-indent": [2, 4]
+    },
+    "globals": {
+        "describe": false,
+        "it": false,
+        "expect": false,
+        "shallow": false
+    }
+}
```
Comment (ribeiroguilherme) [inline .eslintrc.json]: Might makes sense to remove `.json` from the file name, so we can have consistency with other config files

File: package.json [hunk 2]
```diff
@@ -7,6 +7,7 @@
     "build": "webpack --config ./etc/webpack/webpack.dev.config.js",
     "build:prod": "webpack --config ./etc/webpack/webpack.prod.config.js",
     "start": "webpack-dev-server --config ./etc/webpack/webpack-storybook.dev.config.js",
+    "lint": "eslint .",
```
Comment (ribeiroguilherme) [inline package.json]: During the linting execution seems to have some error at `webpack-storybook.dev.config.js`

File: etc/webpack/utils/utils.js [hunk 3]
```diff
@@ -1,17 +1,15 @@
-
 function getRuleJS(...includePaths) {
     return {
         test: /\.js$/,
         include: includePaths,
         use: [
             {
-                loader: 'babel-loader',
+                loader: 'babel-loader'
```
Comment (ribeiroguilherme) [inline etc/webpack/utils/utils.js]: Might be good idea to add `eslint-loader` here, so during the build process or dev mode, we can catch errors easily

File: .eslintrc.json (hunk 1 above)
Comment (eszthoff) [reply .eslintrc.json]: sorry. that is deprecated
see: https://eslint.org/docs/user-guide/configuring#configuration-file-formats

Comment (ribeiroguilherme) [reply .eslintrc.json]: Oh ok!

File: .prettierrc [hunk 4]
```diff
@@ -0,0 +1,6 @@
+{
+    "printWidth": 100,
+    "singleQuote": true,
+    "tabWidth": 4,
+    "useTabs": false
```
[the remark this answers is not in the document]
Comment (eszthoff) [reply .prettierrc]: removed

File: etc/webpack/utils/utils.js (hunk 3 above)
Comment (eszthoff) [reply etc/webpack/utils/utils.js]: added for prod but not for dev

**Pull request merged**